src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpGoToGlobalImportsTests.cs

```diff
@@ -21,7 +21,7 @@ public CSharpGoToGlobalImportsTests() : base(solutionName: nameof(CSharpGoToGlob
         {
         }
 
-        [IdeFact]
+        [IdeFact(Skip = "https://github.com/dotnet/roslyn/issues/72018")]
```

Comment:
reported flakey tests.  disabling in a separate PR that's also merged into this one.